docs/pages/setup/reference/cli.mdx

```diff
@@ -1289,6 +1289,68 @@ Print cluster version:
 tctl version
 ```
 
+## Resource Filtering
+
+Both `tsh` and `tctl` allow you to filter node, app, db, and kube resources using the `--search` and `--query` flags.
+
+The `--search` flag performs a simple fuzzy search on resource fields. For example, `--search=mac` searches for resources containing `mac`.
+
+The `--query` flag allows you to perform more sophisticated searches using a [predicate language](#predicate-language) that is detailed in the next section.
+
+In both cases, you can further refine the results by appending a list of comma-separated labels to the command. For example:
+
+```bash
+$ tsh ls --search=foo,bar labelKey1=labelValue1,labelKey2=labelValue2
+```
+
+### [Predicate Language](#predicate-language)
+
+The `--query` flag uses the same predicate language used else where but tailored for resources. 
```

Comment:
```suggestion
The `--query` flag uses the same predicate language used elsewhere but tailored for resources. 
```

I'm curious if @ptgott has suggestions on improving this. I don't think we should expect readers to know where "elsewhere" is, but I'm drawing a blank on better options.